bot/role_management/role_management.py

```diff
@@ -297,7 +298,7 @@ async def reaction_role_add(self, payload: discord.RawReactionActionEvent):
         Args:
             payload (discord.RawReactionActionEvent): The payload for the triggered event.
         """
-        if payload.channel_id == self.ch_role.id and not payload.member.bot:
+        if not payload.member.bot and payload.channel_id == self.ch_role.id:
```

Comment:
Why switch these two?

Reply:
Yeah, the other way around is probably better.